### ISSUE TITLE
Improve watchman error text

### ISSUE THREAD

**Issue by felixbuenemann** [ember-cli/ember-cli#3458]
ember-cli complains about watchman not being installed, without providing helpful steps to fix this:
```bash
ember -v
version: 0.2.0-beta.1
Could not find watchman, falling back to NodeWatcher for file system events
node: 1.4.3
npm: 2.1.8
```

As an ember newb this is very confusing. What is watchman? Why wasn't it installed by npm? So my first try was `npm install -g watchman`, which worked, but installed a completely different watchmen. Now ember-cli complained that watchman was found, but incompatible:
```bash
ember -v
version: 0.2.0-beta.1
Looks like you have a different program called watchman, falling back to NodeWatcher
node: 1.4.3
npm: 2.1.8
```

My thoughts: Hmm, maybe I need an older version of the npm package, but which one?

Only after some more googling I finally found out that I was supposed to install watchman via homebrew.

**So why not put a link to the watchman homepage or installation instructions into the message?**

Example:
```bash
ember -v
version: 0.2.0-beta.1
Could not find watchman, falling back to NodeWatcher for file system events
For better performance install watchman from https://facebook.github.io/watchman/
node: 1.4.3
npm: 2.1.8
```

**Comment by felixbuenemann** [comment]
I'm happy to come up with a PR, if there is consensus about what the message should look like.

**Comment by rwjblue** [comment]
@felixbuenemann - Agreed on both points: 

* only show on platforms that are supported by watchman
* make the message actionable via helpful link and description

**Comment by stefanpenner** [comment]
we should link to something we control

* http://ember-cli.com/watchman

it should likely contain:

* what is watchman
* how to get watchman
* troubleshooting watchman

**Comment by felixbuenemann** [comment]
@stefanpenner Where are the sources for the ember-cli website or who is responsible for updating it?

**Comment by stefanpenner** [comment]
@felixbuenemann https://github.com/ember-cli/ember-cli/tree/gh-pages currently @joostdevries and @chnn have been stewarding it, and working on a refresh.

**Comment by felixbuenemann** [comment]
@stefanpenner For now I've added a small section on watchman to the getting started section (see PR #3543). I think we shouldn't duplicate any info that is just as well covered by the watchman github page.

**Comment by stefanpenner** [comment]
@felixbuenemann thanks, we will keep this open until we link from the error to this.

**Comment by felixbuenemann** [comment]
@stefanpenner See #3544 for those changes.

**Comment by rwjblue** [comment]
Fixed by #3544
